### PR TITLE
Editing pass through main text

### PR DESCRIPTION
**Pull request by jeromekelleher**
Mainly just deleting a few words and redundant sentences.

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -99,11 +99,6 @@
 We provide thoroughly tested implementations of Demes parsers in Python
 and C, and showcase initial support in a number of simulators and inference
 methods.
-
```
Comment (jeromekelleher) [inline paper.tex:102]: I think we're splitting hairs here - we showcase them by mentioning them in the table and we also definitely showcase moments.

**Review by grahamgower — APPROVED** [review]
LGTM

File: paper.tex (hunk 1 above)
Comment (grahamgower) [reply paper.tex:102]: I read this as "we're going to show you how to do some parameter inference using Demes". But :shrug:

Comment (jeromekelleher) [reply paper.tex:102]: I see - maybe you could tweak the language a bit there then?

**Review by apragsdale — APPROVED** [review]

**Pull request merged**